Neos.ContentRepository/Classes/Domain/Projection/Content/TraversableNodeInterface.php

```diff
@@ -53,9 +53,9 @@ public function getDimensionSpacePoint(): DimensionSpacePoint;
 
     /**
      * Retrieves and returns the parent node from the node's subgraph.
-     * Returns null if this is a root node.
+     * If no parent node is present, an {@link NodeException} is thrown.
```

Comment:
```suggestion
     * If no parent node is present, a {@link NodeException} is thrown.
```

Reply:
Let's merge this as is and fix the typo in a follow-up? Can't commit the suggestion into this fork branch